clap_derive/Cargo.toml

```diff
@@ -1,13 +1,10 @@
+cargo-features = ["workspace-inheritance"]
+
 [package]
 name = "clap_derive"
-version = "3.1.18"
-edition = "2018"
-include = [
-	"src/**/*",
-	"Cargo.toml",
-	"LICENSE-*",
-	"README.md"
-]
+version.workspace = true
```

Comment:
FYI Before, we only could enforce `clap` and `clap_derive` shared a version via `cargo-release` but now we can do it here.